tests/test_stub_utils.py

```diff
@@ -0,0 +1,329 @@
+import pytest
+import numpy as np
+import numpy.testing as npt
+
+from pyfar import Signal
+
+import stub_utils
```

Comment:
```suggestion
from . import stub_utils
```

Reply:
I get the warning `Attempted relative import beyond top-level package` thrown for doing it that way, also VSCode does not detects the tests anymore... Any propose on whats better?